src/Optimization.cmake

```diff
@@ -32,3 +32,13 @@ macro(enable_native_optimization project_name)
     endif()
   endif()
 endmacro()
+
+macro(disable_exceptions project_name)
+  target_compile_options(${project_name} INTERFACE $<$<CXX_COMPILER_ID:MSVC>:/EHs-c- /D_HAS_EXCEPTIONS=0>)
+  target_compile_options(${project_name} INTERFACE $<$<NOT:$<CXX_COMPILER_ID:MSVC>>:-fno-exceptions>)
```

Comment:
Is it enough to add only `-fno-exceptions` or should `-fno-unwind-tables` also be added?

Reply:
Seems fine for me ... https://alex-robenko.gitbook.io/bare_metal_cpp/compiler_output/exceptions

_(I personally use this flags for some embedded and [ETL](https://www.etlcpp.com/) stuff :P)_